src/adapters/sqlite/makeDispatcher/node/Database.js

```diff
@@ -0,0 +1,129 @@
+// @flow
+const fs = require(`fs`)
+const SQliteDatabase = require('better-sqlite3')
+
+class Database {
```

Comment:
don't call it `Database`, since that's already a class name for something else (it's not _technically_ a conflict, but it may be confusing to someone in the future)

Reply:
Again, just following Swift :/

Reply:
I just checked and Android uses the same structure...